internal/cli/kraft/cloud/deploy/deploy.go

```diff
@@ -159,8 +161,13 @@ func (opts *DeployOptions) Run(ctx context.Context, args []string) error {
 
 	// Preflight check: check if `--name` is already taken:
 	if len(opts.Name) > 0 {
-		if _, err := opts.Client.Instances().GetByNames(ctx, opts.Name); err == nil {
-			return fmt.Errorf("service name '%s' is already taken", opts.Name)
+		var resp *kcclient.ServiceResponse[kcinstances.GetResponseItem]
```

Comment:
This declaration seems unnecessary. Why not simply take the `resp, err :=` assignment out of the `if` statement below?

Reply:
stop speaking sense 😅